src/App.css

```diff
@@ -1,39 +1,67 @@
-.App {
-  text-align: center;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.conainter {
+  width: 100%;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.calculator-container {
+  width: 100%;
+  height: 100%;
   display: flex;
   flex-direction: column;
+}
+
+.calculator-header {
+  width: 100%;
+  background: #a9a9a9;
+  padding: 20px 6px;
+  font-size: 24px;
+  font-weight: 600;
+  color: #fff;
+  display: flex;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  justify-content: flex-end;
+}
+
+.item-container {
+  display: grid;
```

Comment:
- [ ]  Your design should match to the one provided from the template as it seemed that you some how have color different such as some button texts are white while they were supposed to be black, and the input area should  be a little me gray. Implement this so that your design can match exact to the one provided from the design template. 

Your design 
![image](https://user-images.githubusercontent.com/43877689/227342524-571ecef1-e855-492a-b9bc-ef024e76c03a.png)

template design 
![image](https://user-images.githubusercontent.com/43877689/227342790-57b7e6b9-e082-4878-9e27-54ec37aabf28.png)